Index: pathauto.module
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/pathauto/pathauto.module,v
retrieving revision 1.25
diff -u -r1.25 pathauto.module
--- pathauto.module	30 Aug 2006 20:38:43 -0000	1.25
+++ pathauto.module	14 Sep 2006 14:19:28 -0000
@@ -66,6 +66,12 @@
     '#description' => t('Character used to separate words in titles. This will replace '.
       'any spaces and punctuation characters.'));
 
+  $form['general']['pathauto_quotes'] = array('#type' => 'radios',
+    '#title' => t('Quotation marks'),
+    '#default_value' => variable_get('pathauto_quotes', 0),
+    '#options' => array(t('Remove'), t('Replace by separator')),
+    );
+    
   $form['general']['pathauto_max_length'] = array('#type' => 'textfield',
     '#title' => t('Maximum alias length'), '#size' => 3, '#maxlength' => 3,
     '#default_value' => variable_get('pathauto_max_length', 128),
@@ -254,7 +260,10 @@
     'ß'=>'ss','ſ'=>'ss');
 
   // Drop apostrophes
-  $output = str_replace("'", "", $string);
+  $separator = variable_get('pathauto_separator', '_');
+  $quotes = variable_get('pathauto_quotes', 0);
+  $output = str_replace("'", ($quotes ? $separator : ''), $string);
+  
   // Convert accented characters to their ASCII counterparts...
 /*  $output = strtr(utf8_decode($output),
        "\xA1\xAA\xBA\xBF".
@@ -271,7 +280,6 @@
 
   // Preserve alphanumerics, everything else becomes a separator
   $pattern = '/[^a-zA-Z0-9]+/ ';
-  $separator = variable_get('pathauto_separator', '_');
   $output = preg_replace($pattern, $separator, $output);
 
   // Trim any leading or trailing separators (note the need to
